Doc restructure (#1551)
openvinotoolkit/model_server: docs/build_from_source.md

# Building Container from Source {#ovms_docs_build_from_source}

Before starting the server, make sure your hardware is [supported](https://docs.openvino.ai/2022.2/_docs_IE_DG_supported_plugins_Supported_Devices.html) by OpenVINO.

> **NOTE**: OpenVINO Model Server execution on baremetal is tested on Ubuntu 20.04.x. For other operating systems, starting model server in a [docker container](./docker_container.md) is recommended.
   
1. Clone model server git repository.
2. Navigate to the model server directory.
3. Use a precompiled binary or build it in a Docker container.
4. Navigate to the folder containing the binary package and unpack the `tar.gz` file.

Run the following commands to build a model server Docker image:

```bash

git clone https://github.com/openvinotoolkit/model_server.git

cd model_server   
   

> # automatically build a container from source
> # it places a copy of the binary package in the `dist` subfolder in the Model Server root directory
> make docker_build
> 
> # unpack the `tar.gz` file
> cd dist/ubuntu && tar -xzvf ovms.tar.gz

Reviewer: ```suggestion
# replace to 'redhat` if using UBI base image
export BASE_OS=ubuntu

# automatically build a container from source
# it will also place a copy of the binary package in the `dist` subfolder in the Model Server root directory
make docker_build BASE_OS=${BASE_OS}

# unpack the `tar.gz` file
cd dist/${BASE_OS} && tar -xzvf ovms.tar.gz
```
Can we add BASE_OS env like previously in host.md? Path dist/ubuntu can change depending on OS used
